flyway/migrations/V3__Create_pets_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE Pets (
+    PetID uniqueidentifier DEFAULT NEWSEQUENTIALID() PRIMARY KEY,
+    ClientID_U_FK uniqueidentifier NOT NULL,
+    PetName varchar(255) NOT NULL,
+    SpeciesID_U_FK uniqueidentifier NOT NULL,
+    SpecialRequirements varchar(MAX),
+    AdditionalNotes varchar(MAX),
+    YearOfBirth int,
+    FOREIGN KEY (ClientID_U_FK) REFERENCES Users(UserID),
+    FOREIGN KEY (SpeciesID_U_FK) REFERENCES Species(SpeciesID)
```

Comment:
Dont forget to change the SpeciesID naming convention if you change the one in the Species table